x-pack/plugins/serverless_search/public/application/components/overview.tsx

```diff
@@ -70,6 +71,16 @@ export const ElasticsearchOverview = () => {
     apiKey: clientApiKey,
     cloudId,
   };
+  const { hash } = useLocation();
+  useEffect(() => {
+    if (hash) {
+      const id = hash.replace('#', '');
+      const element = document.getElementById(id);
+      if (element) {
+        element.scrollIntoView({ behavior: 'smooth' });
```

Comment:
would this work multiple times if we re-render this page for some reason ?
I think it `#links`may work without javascript even does it ?

Reply:
After some research, I think it is fine with your approach, just make sure we trigger this effect once.

Reply:
Unfortunately, `#links` work only if the page is rendered completely. That's the reason why had to go with this approach. 

I have added a change in useEffect to make sure the page is rendered only if the hash value is changed. Commit 1f06f0f7fa3ba6dacba93924d2614113c0fc40de.